tsconfig.json

```diff
@@ -29,12 +25,5 @@
   "exclude": [
     "node_modules",
     "packages/*/dist"
-  ],
-  "ts-node": {
-    // these options are overrides used only by ts-node
-    // same as the --compilerOptions flag and the TS_NODE_COMPILER_OPTIONS environment variable
-    "compilerOptions": {
-      "module": "commonjs"
-    }
-  }
```

Comment:
question: why is this being removed? This was specifically for the CLI

Reply:
As long as the CLI `create-package` still works, this should be fine

Reply:
Oh, I thought it was used for cypress. But since `create-package` seems ok I suggest remove this block to keep tsconfig simple

Reply:
Sounds good